src/main/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepository.kt

```diff
@@ -33,8 +33,8 @@ interface CertificateRepository : JpaRepository<Certificate, UUID> {
 }
 
 object CertificateRepositoryExtensions {
-    fun CertificateRepository.findByPrintRequestStatusAndBatchId(status: Status, batchId: String): List<Certificate> {
-        return findByPrintRequestsBatchId(batchId)
+    fun CertificateRepository.findDistinctByPrintRequestStatusAndBatchId(status: Status, batchId: String): List<Certificate> {
+        return findByPrintRequestsBatchId(batchId).toSet()
```

Comment:
Without this there are duplicate certificates returned that caused bug with caused Zip file not to be produced.